pkg/pulumiyaml/sort.go

```diff
@@ -256,3 +248,11 @@ func isConfigNodeEnv(n graphNode) bool {
 	_, ok := n.(configNodeEnv)
 	return ok
 }
+
+func stripConfigNamespace(n, s string) string {
+	tok := strings.Split(s, ":")
+	if len(tok) == 2 && tok[0] == n {
+		return tok[1]
+	}
+	return s
```

Comment:
AFAK config keys must be `SEG1:SEG2`. We will never have a scenario where `len(tok) > 2`. This lets us simplify this block to `return strings.TrimPrefix(n+":", s)`.